src/locale/locales/fr/messages.po

```diff
@@ -1756,11 +1737,11 @@ msgstr "Bibliothèque"
 
 #: src/view/screens/Settings.tsx:460
 msgid "Light"
-msgstr ""
+msgstr "Clair"
 
 #: src/view/com/util/post-ctrls/PostCtrls.tsx:189
 msgid "Like"
-msgstr ""
+msgstr "Aimer"
```

Comment:
It does not seem to be translated elsewhere. So maybe keep "Like"?

Reply:
Yep, let's """translate""" this by `Liker` for now 👍